lib/har/stats.rb

```diff
@@ -136,10 +136,12 @@ def calculate_server_time
     end
 
     def get_source_url
-      pages.first.is_redirect? ? entries[1].request.url.slice(0, 255) : entries.first.request.url.slice(0, 255)
+      pages.first.is_redirect? ? entries[1]&.request&.url&.slice(0, 255) : entries.first.request.url.slice(0, 255)
```

Comment:
Should you also add safe nav to the other side of the conditional?

Reply:
Yeah. I'm not sure if we would get to this point if there were no requests. But I don't think it could hurt.